packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -116,7 +116,7 @@ describe('Create new sdk command', () => {
     expect(() =>
       packageManagerExec({
         script: 'schematics',
-        args: ['@ama-sdk/schematics:typescript-core', '--spec-path', path.join(path.relative(sdkPackagePath, sdkFolderPath), 'swagger-spec.yml')]
+        args: ['@ama-sdk/schematics:typescript-core', '--spec-path', path.join(path.relative(sdkPackagePath, sdkFolderPath).split(path.sep).join(path.posix.sep), 'swagger-spec.yml')]
```

Comment:
Not sure to understand this change because the `path.join` will retransform to `\` on Windows.
Maybe you wanted to change it to `path.posix.join` as well

Reply:
I think I don't need `.split(path.sep).join(path.posix.sep)` at all here. It should be OS specific.